app/customer.py

```diff
@@ -0,0 +1,67 @@
+import math
+import datetime
+from app.shop import Shop

```

Comment:
![Screenshot from 2023-10-13 13-38-45](https://github.com/mate-academy/py-shop-trip/assets/122517203/005a358e-97f2-465f-9d3a-a1776ddcf110)

fix imports according to PEP8